week2/homework/src/bonus/actions/_update.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const Util = require('../helpers/Util');
+
+async function update(filePath, line, newTodo) {
+  /* eslint brace-style: 2 */
+  try {
+    const todosJSON = await Util.getJSONFromFile(filePath);
+
+    const todoList = todosJSON.todos;
+
+    if (isNaN(line)) {
+      return console.log(`'${line}' is not an expected number value!`);
+    }
+
+    if (todoList.length === 0) {
+      return console.log('The file is already empty!');
+    }
+
+    if (line > todoList.length) {
+      return console.log(
+        `You can't update line '${line}'. Maximum line number is '${todoList.length}'`
+      );
+    }
+
+    const oldTodo = todoList[line - 1];
+
+    todoList.splice(line - 1, 1, newTodo);
```

Comment:
can you think of a simpler way to do this?

Reply:
🤦‍♂‍‍
```javascript
todoList[line - 1] = newTodo;
```